src/interface/web/tailwind.config.ts

```diff
@@ -2,6 +2,10 @@ import type { Config } from "tailwindcss"
 
 const config = {
     safelist: [
+        {
+        pattern: /to-(blue|yellow|green|pink|purple|orange|red)-(50|100|200|950)/,
```

Comment:
We'd have to extend this to include all the viable colors options. See the Agent's model color file to see the list.